TypeCobol.Test/Parser/Programs/Cobol85/DataRenames.rdz.cbl

```diff
@@ -0,0 +1,16 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. PGM1.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 TEST-GROUP1.
+          05 TEST-SGROUP1.
+             10 TEST-ELT1 PIC X(04).
+             10 TEST-ELT2 PIC X(350).
+          05 TEST-SGROUP2.
+             10 TEST-ELT3 PIC X(200).
+             10 TEST-ELT4 PIC X(1000).
+          05 TEST-SGROUP5.
+             10 TEST-ELT5 PIC X(100).
+          66 TEST-RENAMES RENAMES TEST-ELT2
+                          THRU    TEST-ELT4.
+       END PROGRAM PGM1.
```

Comment:
Ok for this test. It would be good to test also with the following sample code
```cob
       01 TEST-GROUP.
          05 TEST-SGROUP1.
             10 TEST-ELT1 PIC X.
             10 TEST-ELT2 PIC X.
          05 TEST-SGROUP2.
             10 TEST-ELT3 PIC X.
             10 TEST-ELT4 PIC X.
             10 TEST-ELT5 PIC X.
             10 TEST-ELT6 PIC X.
             66 TEST-RENAMES RENAMES TEST-SGROUP2
```

Reply:
See aad21509a1be318362a785781affcd82f74fd35e.